.github/workflows/ci.yml

```diff
@@ -47,6 +47,20 @@ jobs:
     - name: Fuzz Test
       run: node Makefile fuzz
 
+  test_on_deno:
+    name: Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOs-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: denoland/setup-deno@v1
+    - name: Install Packages
+      run: npm install
```

Comment:
how is it deno support if it requires npm, which doesn’t ship with deno afaik?

Reply:
In compat mode Deno can understand `package.json` and `node_modules` and apply Node's resolution. Soon Deno will be able to run `npm` too. However, there's currently no plan to ship with `npm` or have built-in `npm` functionality.

Reply:
Right - so how is it "supporting deno" if it doesn't work with it out of the box?

Reply:
@ljharb The `deno run --compat` path is not the only way to use a package written originally for Node/NPM in Deno. CDNs like https://esm.sh and https://cdn.skypack.dev will replace Node imports with direct url imports to `https://deno.land/std/node` on the fly for example, which would make running eslint as easy as `deno run --allow-read --allow-write --allow-env --no-check https://esm.sh/eslint`. Or install it with `deno install --allow-read --allow-write --allow-env --no-check https://esm.sh/eslint`. So no NPM required. (Note both esm.sh and cdn.skypack.dev have unrelated issues that cause this to not work yet, but the theory is there :smiley:).

I think this discussion is unrelated to the PR though, so probably best to move it to a different venue (maybe a GH Discussion on denoland/deno?).